index.html

```diff
@@ -17,128 +18,277 @@
+                  <li><a href="#"><img src="Images\Vector1.svg" alt="" width="19.99px" height="19.96px"></a></li>
+                  <li><a href="#"><img src="Images\Vector3.svg" alt="" width="20px" height="16px"></a></li>
                </ul>
             </div>
-            <div class="card-1-btn"><button>See Project</button></div>
-        </section>
-    </section>
-    <section class="ams">
-      <h2>About Myself</h2>
-      <p>
-         Hello I'm a software developer! I can help you build a product, feature or website
-         Look through some of my work and experiance! If you like what you see and have a project you need coded,
-         don't hestiate to contact me.
-      </p>
-      <h3 class="letconnect_text"><a href="https://github.com/abiy006">LET'S CONNECT</a></h3>
-      <div class="letconnect_icons">
-         <ul>
-            <li><a href="#"><img src="Images\Vector4.svg" alt="" width="17.99px" height="20px"></a></li>
-            <li><a href="#"><img src="Images\Vector2.svg" alt="" width="20px" height="20px"></a></li>
-            <li><a href="#"><img src="Images\Vector5.svg" alt="" width="14.17px" height="20px"></a></li>
-            <li><a href="#"><img src="Images\Vector1.svg" alt="" width="19.99px" height="19.96px"></a></li>
-            <li><a href="#"><img src="Images\Vector3.svg" alt="" width="20px" height="16px"></a></li>
-         </ul>
-      </div>
-      <button>Get my resume</button>
-   <div class="About-languages">
-         <ul>
-            <li class="about-lang-menu"><a href="#"><button><h3>Languages</h3><img src="Images\about_down_arrow.svg" alt="" width="14px" height="8px"></button></a></li>
+            <div class="see-project-btn"><button>Get my resume</button></div>
+         </div>
+         <div class="about_part2">
             <ul>
-               <li><a href="#"><button><img src="Images\about_javascript2.svg" alt="" width="48px" height="48px"><h3>JavaScript</h3></button></a></li>
-               <li><a href="#"><button><img src="Images\about_html2.svg" alt="" width="48px" height="48px"><h3>HTML</h3></button></a></li>
-               <li><a href="#"><button><img src="Images\about_css2.svg" alt="" width="48px" height="48px"><h3>CSS</h3></button></a></li>
+               <li><a href="#"><img src="Images\Frame 77.svg" alt="" width="90%" height="72px"></a>
+                  <ul>
+                     <li><a href="#"><img src="Images\Frame 9.svg" alt="" width="122px" height="120px"></a></li>
+                     <li><a href="#"><img src="Images\Frame 10.svg" alt="" width="122px" height="120px"></a></li>
+                     <li><a href="#"><img src="Images\Frame 11.svg" alt="" width="122px" height="120px"></a></li>
+                  </ul>
+               </li>
+               <li><a href="#"><img src="Images\Frame 88.svg" alt="" width="90%" height="72px"></a></li>
+               <li><a href="#"><img src="Images\Frame 99.svg" alt="" width="90%" height="72px"></a></li>
```

Comment:
- **[OPTIONAL]** This section should be a combination of text content for some elements and image content for others, currently, all the content is an image exported from the Figma Design, which will make this section not accessible from the screen readers.

### Example of content that is an image
![Screenshot from 2023-03-31 23-46-59](https://user-images.githubusercontent.com/68030297/229245884-f1089afb-03fa-41d7-9a4c-8f26b5fbc844.png)

### Example of content that is a text
![Screenshot from 2023-03-31 23-46-33](https://user-images.githubusercontent.com/68030297/229245847-06c48866-4d46-4f2d-9168-d3cc0384063c.png)